hedera-node/src/test/java/com/hedera/services/txns/token/TokenMintTransitionLogicTest.java

```diff
@@ -98,6 +103,28 @@ void followsHappyPath() {
 		verify(store).persistTokenRelationship(treasuryRel);
 	}
 
+	@Test
+	void followsUniqueHappyPath() {
+		// setup:
+		treasuryRel = new TokenRelationship(token, treasury);
+
+		givenValidUniqueTxnCtx();
+		given(accessor.getTxn()).willReturn(tokenMintTxn);
+		given(txnCtx.accessor()).willReturn(accessor);
+		given(store.loadToken(id)).willReturn(token);
+		given(token.getTreasury()).willReturn(treasury);
+		given(store.loadTokenRelationship(token, treasury)).willReturn(treasuryRel);
+		given(token.getType()).willReturn(TokenType.NON_FUNGIBLE_UNIQUE);
+		// when:
+		subject.doStateTransition();
+
+		// then:
+		verify(token).mint(any(), any(), any(), any());
```

Comment:
can we place actual values here?

Reply:
We can, but the test won't pass as the first argument is created outside the scope. so the reference would be different, e.g. test will fail cuz of different args

Reply:
Maybe we can specify the arguments that we can, f.e treasury rel or serial numbers etc